src/participants/nikoleta.md

```diff
@@ -0,0 +1,5 @@
+# Nikoleta E. Glynatsi
+
+Nikoleta likes video-games and comics.
+
+Her website is here: [nikoleta-v3.github.io](https://nikoleta-v3.github.io/
```

Comment:
Hi @Nikoleta-v3, looks like a typo here, we need a closing bracket at the end of your url:

```
[nikoleta-v3.github.io](https://nikoleta-v3.github.io/)
```